### PR TITLE
Enable memberOf Overlay for first mdb

### PR DESCRIPTION
**Pull request by mrvanes**
Untested. I **hope** this covers the first part of the change.

### CODE REVIEW

File: roles/ldap/tasks/main.yml [hunk 1]
```diff
@@ -78,6 +79,20 @@
   failed_when: "result.rc not in [0,20]"
   changed_when: "result.rc==20"
 
+- name: Enable memberof overlay
+  command: ldapmodify -Q -Y EXTERNAL -H ldapi:/// -f "{{ ldap_ldif_dir }}/enable_memberof.ldif"
+  # return code 20 means entry already exists
+  register: result
+  failed_when: "result.rc not in [0,20]"
+  changed_when: "result.rc==20"
```
Comment (baszoetekouw) [inline roles/ldap/tasks/main.yml]: The `changed_on` doesn't seem to work correctly: the travid bluid shows "OK" on the first pass and "changed" on the second pass.
Please check if this actually works as intended, and if so, why ldapmodify doesn't return 20.

File: roles/ldap/tasks/main.yml [hunk 2]
```diff
@@ -78,6 +79,20 @@
...
+
+- name: Configure memberof overlay for 1mdb
+  command: ldapmodify -Q -Y EXTERNAL -H ldapi:/// -f "{{ ldap_ldif_dir }}/enable_memberof_1mdb.ldif"
+  # return code 20 means entry already exists
+  register: result
+  failed_when: "result.rc not in [0,20]"
+  changed_when: "result.rc==20"
+
```
Comment (baszoetekouw) [inline roles/ldap/tasks/main.yml]: Also doen't seem to work correctly.  Travis shows an "OK" twice.

File: roles/ldap/tasks/main.yml (hunk 1 above)
Comment (mrvanes) [reply roles/ldap/tasks/main.yml]: This is untested. I copied the changed_when condition from surrounding tasks that do exactly the same?

Comment (baszoetekouw) [reply roles/ldap/tasks/main.yml]: Those are probably also not correct.  The ldap task is a mess in this regard.

I think it should be:
```yml
  register: result
  failed_when: "result.rc not in [0,20,68]"
  changed_when: "result.rc not in [20,68]"
```

Volgens https://docs.servicenow.com/bundle/london-platform-administration/page/administer/reference-pages/reference/r_LDAPErrorCodes.html is de betekenis:
 - 0: ok
 - 20: attribuut bestaat al
 - 68: entry bestaat al

Comment (mrvanes) [reply roles/ldap/tasks/main.yml]: Done

**Pull request merged**